internal/k8s/leader_election_callbacks.go

```diff
@@ -15,24 +15,38 @@
 package k8s
 
 import (
+	"sync"
+
 	v1 "k8s.io/api/core/v1"
 
+	avicache "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/cache"
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/lib"
+	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/rest"
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/utils"
 )
 
+var once sync.Once
+
 func (c *AviController) OnStartedLeading() {
 	lib.AKOControlConfig().SetIsLeaderFlag(true)
 	utils.AviLog.Debugf("AKO became a leader")
 	lib.AKOControlConfig().PodEventf(v1.EventTypeNormal, "LeaderElection", "AKO became a leader")
 	c.publishAllParentVSKeysToRestLayer()
+
+	once.Do(c.cleanupStaleVSes)
```

Comment:
@swathinsankaran: Should we clean up stale vs first and then populate vskeys?

Reply:
As discussed the order should not matter as the stale entries are populated from the virtual service cached during bootup.